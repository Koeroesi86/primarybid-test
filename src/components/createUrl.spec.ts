import { render } from '@testing-library/vue';
import CreateUrl from './createUrl';

describe('CreateUrl', () => {
  it('should render', () => {
    const result = render(CreateUrl);

    expect(result.container).toMatchSnapshot();
  });
});
