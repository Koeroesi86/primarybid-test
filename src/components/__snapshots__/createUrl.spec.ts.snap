// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateUrl should render 1`] = `
<div>
  <div
    class="createUrl"
  >
    <input
      placeholder="Shorten some urls"
      type="url"
    />
     
    <button
      type="button"
    >
      
        Shorten
      
    </button>
  </div>
</div>
`;
